src/lib_ccx/ccx_encoders_webvtt.c

```diff
@@ -492,7 +492,8 @@ int write_cc_buffer_as_webvtt(struct eia608_screen *data, struct encoder_ctx *co
 				}
 
 				// write current text symbol
-				write(context->out->fh, &(context->subline[j]), 1);
+				if (context->subline[j] != '\0')
+					write(context->out->fh, &(context->subline[j]), 1);
```

Comment:
Is this correct, shouldn't we write a space instead?

Reply:
There are the same number of spaces in the srt as the post patch vtt. It seems to me the null char is some residue from pointer handling.